be/src/olap/rowset/beta_rowset_reader.cpp

```diff
@@ -304,10 +304,12 @@ Status BetaRowsetReader::next_block_view(vectorized::BlockView* block_view) {
 
 bool BetaRowsetReader::_should_push_down_value_predicates() const {
     // if unique table with rowset [0-x] or [0-1] [2-y] [...],
-    // value column predicates can be pushdown on rowset [0-x] or [2-y], [2-y] must be compaction and not overlapping
+    // value column predicates can be pushdown on rowset [0-x] or [2-y], [2-y]
+    // must be compaction, not overlapping and don't have sequence column
     return _rowset->keys_type() == UNIQUE_KEYS &&
```

Comment:
If _should_push_down_value_predicates returns false, then the value predicates is not evaluated in segment iterator. Then when are these value predicates evaluated？

Reply:
These value predicates will be evaluated by scan node.